src/promptflow-tools/requirements.txt

```diff
@@ -1,2 +1,7 @@
 google-search-results==2.4.1
-promptflow
+
+# TODO: replace package after publish the pf package which support openai 1.x
```

Comment:
append promptflow dir to pythonpath, then we can consume main branch code in tool tests ci.